improver_tests/ensemble_copula_coupling/test_RebadgePercentilesAsRealizations.py

```diff
@@ -93,6 +93,39 @@ def test_raises_exception_if_realization_already_exists(self):
         with self.assertRaisesRegex(InvalidCubeError, msg):
             Plugin().process(self.cube)
 
+    def test_raises_exception_if_percentiles_unevenly_spaced(self):
+        """Check that we raise an exception if the input percentiles are not
+        evenly spaced."""
+        self.cube = set_up_percentile_cube(
+            np.sort(ECC_TEMPERATURE_REALIZATIONS.copy(), axis=0),
+            np.array([25, 50, 90], dtype=np.float32),
+        )
+        msg = r"The percentile cube provided cannot be rebadged as ensemble realizations.*"
+        with self.assertRaisesRegex(ValueError, msg):
+            Plugin().process(self.cube)
```

Comment:
I think in all the three added tests, this could just be `cube`, rather than `self.cube`, as this variable is only used locally within this method.

```suggestion
        cube = set_up_percentile_cube(
            np.sort(ECC_TEMPERATURE_REALIZATIONS.copy(), axis=0),
            np.array([25, 50, 90], dtype=np.float32),
        )
        msg = r"The percentile cube provided cannot be rebadged as ensemble realizations.*"
        with self.assertRaisesRegex(ValueError, msg):
            Plugin().process(cube)
```

Reply:
Changed `self.cube` to `cube` in all three new tests.